Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG GOARCH="amd64"
```

Comment:
We are fixing the architecture which breaks k8s cluster bootstrapping on T2A or C4A instances. cc @aojea